src/queues.py

```diff
@@ -8,7 +8,13 @@
 try:
     from multiqueue import MultiQueue
 except ImportError:
-    from .multiqueue import MultiQueue
+    try:
+        from .multiqueue import MultiQueue
+    except ImportError as e:
+        import sys
+        if 'bitmessagemock' not in sys.modules:
+            raise e
+        from six.moves.queue import Queue as MultiQueue
```

Comment:
MultiQueue = queue.Queue ?

Reply:
As discussed with @PeterSurda, we have just monkey patched it for Kivy-mock.